.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
```

Comment:
What does this do?

Reply:
It's gitignored but that file should be installed for you locally as well, after a `yarn install`, here it is:
https://github.com/typicode/husky/blob/3d1124185908a9b8fd21d349c6bedbf84b29a1e6/packages/husky/src/husky.sh

looks like: logging, config loading, debug stuff